llpc/translator/lib/SPIRV/SPIRVReader.h

```diff
@@ -315,6 +316,23 @@ class SPIRVToLLVM {
                                              std::function<Value *(Value *)> createImageOp);
 
   Function *createLibraryEntryFunc();
+
+  const Vkgc::PipelineOptions *getPipelineOptions() const;
+
+  // This flag must explicitly be updated while translating if we want to continue inserting at the last insert point
+  // instead of omitting it.
+  bool continueAtLastInsertPoint = false;
+
+  template <class MemOpType> SPIRVValue *getValueForScratchBoundsCheckWrapper(MemOpType *toBeWrapped) const;
+  struct ScratchBoundsCheckInfo {
+    bool shouldInsertBoundsCheck;
+    BasicBlock *entryBlock;
+    BasicBlock *intermediateBlock;
```

Comment:
Could you add comments explaining what each field is for? For example, the intermediate block does performs a memory access (right?).

Reply:
Yeah, the intermediate block contains the memory access. This is now described.